docs/ops/condition/Select_1.md

```diff
@@ -17,26 +17,31 @@
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
-  * **Type**: string
+    * *none* - no auto-broadcasting is allowed, all input shapes must match,
+    * *numpy* - numpy broadcasting rules, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md),
+    * *pdpd* - PaddlePaddle-style implicit broadcasting, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md).
+  * **Type**: `string`
   * **Default value**: "numpy"
   * **Required**: *no*
 
 
 **Inputs**:
 
-* **1**: `cond` tensor with selection mask of type `boolean`. The tensor can be 0D.
+* **1**: `cond` tensor of type *T_COND* and arbitrary shape with selection mask.
```

Comment:
I would use either dot "." or hyphen "-" after input name and start description with: A tensor of type...
Same as in [NormalizeL2](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/normalization/NormalizeL2_1.md) for instance

Reply:
Done in this [commit](https://github.com/openvinotoolkit/openvino/pull/6488/commits/22ec1107bcfa18991452f79637c5215ac9d94298).